# Large-scale-RNA-Seq-Transcriptome-Analysis-of-4043-Cancers-and-548-Normal-Tissue-Controls-
### Authors (@slack): 
### Rahma Mamdouh (rahma mam2000) (radwamamdouh580@gmail.com), 
### Alaa Khaled (Alaa043)( alaakhaledexo33@gmail.com), 
### Aya Elhagry (AyaElhagry)( ayaelhagry12@gmail.com), 
### Marwan Magdy (Marwan16)( marwanmagdy.bt@gmail.com) 

## Introduction 
### Individual gene alterations are frequently the focus of cancer research, however this method can overlook the intricate relationships between several genes. Examining gene expression in a variety of malignancies provides a more comprehensive perspective through large-scale RNA-Seq transcriptome research. This work evaluated RNA-Seq data from 4043 cancer samples and 548 normal tissue controls from The Cancer Genome Atlas (TCGA), spanning 12 cancer types. The objective was to catalogue variations in gene expression that are unique to each kind of cancer and investigate the relationship between expression and gene mutations.

## Methodology 
### To compare the transcriptomes of cancer and normal tissues, the study used gene-level differential expression (DE) analysis to perform RNA-Seq analysis of 12 cancer types. Furthermore, gene set association analysis was performed to find gene sets that were consistently changed in all types of cancer. Finding patterns in gene expression and comprehending how these can influence the emergence of cancer was the aim.

## Results
### The DE analysis found that less than 20% of altered genes were common to most cancer types, indicating that many gene expression changes are cancer-specific. However, two lung cancer subtypes, Lung Adenocarcinoma (LUAD) and Lung Squamous Cell Carcinoma (LUSC), were discovered to share 55% of their DE genes, indicating strong molecular similarities. Four major gene clusters (CLUSTER241, CLUSTER514, CLUSTER1011, and CLUSTER932) were consistently altered across seven cancer types, including breast, colon, and bladder cancers, indicating shared pathways between these diseases. On the other hand, there is less than 10% DE gene overlap across kidney malignancies KICH and KIRC, indicating genetic variability even within the same organ cancers.

## Discussion
### The results indicate that changes in gene expression particular to cancer are frequent, suggesting that tailored treatment aimed at these unique profiles could be more successful. The substantial DE gene overlap between LUAD and LUSC points to common biological pathways that may be therapeutically targeted. But kidney tumors are distinct, which emphasizes the need for individualized care. 

## In summary 
### The findings imply that cancer-specific gene expression changes are common and that tailored treatments aimed at these distinct profiles may be more successful. The substantial DE gene overlap between LUAD and LUSC points to common biological pathways that may be therapeutically targeted. But the unique characteristics of kidney tumors draw attention to the necessity of specialized treatment. 

## Video's link 
### https://www.linkedin.com/posts/aya-elhagry-5b6249262_i-am-thrilled-to-share-my-task-of-stage-1-activity-7237801573463015425-VxzY?utm_source=share&utm_medium=member_desktop


